pom.xml

```diff
@@ -241,6 +241,18 @@
 				<role>Mentor</role>
 			</roles>
 		</developer>
+		<developer>
+			<id>solodye</id>
+			<name>Wang, YiZheng</name>
+			<email>xiblade[at]163.com</email>
+			<url>https://github.com/Solodye</url>
+			<organization>Hebei University of Technology</organization>
+			<organizationUrl>http://www.hebut.edu.cn/</organizationUrl>
+			<roles>
+				<role>Committer</role>
```

Comment:
ASF project committer and contributors are different.
https://incubator.apache.org/guides/ppmc.html#adding_new_committers

Long term contributors became committer after project management committees (PMC) votes.

So, removed this line.